omod/src/main/java/org/openmrs/module/openconceptlab/web/rest/resources/ImportResource.java

```diff
@@ -173,4 +180,25 @@ private UpdateScheduler getUpdateScheduler() {
     private static ImportService getImportService() {
         return Context.getService(ImportService.class);
     }
+
+    @Override
+    public Object upload(MultipartFile multipartFile, RequestContext requestContext) throws ResponseException, IOException {
+
+        File file = new File(multipartFile.getOriginalFilename());
+        multipartFile.transferTo(file);
+        ZipFile zipFile = new ZipFile(file);
+        InputStream inputStream = Utils.extractExportInputStreamFromZip(zipFile);
+        file.delete();
```

Comment:
Did you verify the file is deleted? Most likely you open up a stream and OS blocks the file for read/write until you close it. The delete operation will not take effect.